formbricks/formbricks
# update email noti subject

## What does this PR do?

Adds the user email to the subject of the email notification so that not all notifications are gathered in one conversation in Gmail.

## Type of change




## How should this be tested?

Create an in-app survey
Run it for a person without email - check subject line
Run it for a person with email - check subject line

![grafik](https://github.com/formbricks/formbricks/assets/72809645/95e8afa8-b613-4e36-b391-6473ecf1bb4b)


## Checklist
